Can't rebuild joss paper

just in case you forgot, the current state of the paper is here (last commit from me trying podman since docker was failing):

```
[deco]~/heudiconv ❱ git rev-parse HEAD
a6e87d432d26554b17abe924a053205ddf7f8169
[deco]~/heudiconv ❱ git branch
  master
* paper-joss
```

Sadly, both still fail, any ideas why that might be? @asmacdo @yarikoptic 
 
```
[deco]~/heudiconv ❱ make paper OCI_BINARY=podman
podman run --rm --volume /home/chymera/heudiconv:/data:Z --user 1000:1000 --env JOURNAL=joss openjournals/inara
Error: short-name "openjournals/inara" did not resolve to an alias and no unqualified-search registries are defined in "/etc/containers/registries.conf"
make: *** [Makefile:5: paper] Error 125
[deco]~/heudiconv ❱ make paper
docker run --rm --volume /home/chymera/heudiconv:/data:Z --user 1000:1000 --env JOURNAL=joss openjournals/inara
docker: Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error setting cgroup config for procHooks process: bpf_prog_query(BPF_CGROUP_DEVICE) failed: invalid argument: unknown.
make: *** [Makefile:5: paper] Error 125
```

Ok, maybe docker now needs kernel things it didn't need before...

As for podman, ok, that worked, but there's another issue. Any idea what that might be about?

```console
[deco]~/src/heudiconv ❱ make paper OCI_BINARY=podman
podman run --rm --volume /home/chymera/src/heudiconv:/data:Z --user 1000:1000 --env JOURNAL=joss docker.io/openjournals/inara
Trying to pull docker.io/openjournals/inara:latest...
Getting image source signatures
Copying blob ef5531b6e74e done   |
Copying blob 4e821479e4b5 done   |
Copying blob c6b7e74dbfd1 done   |
Copying blob 13d63ff5f1df done   |
Copying blob 00f1e069c9e8 done   |
Copying blob 73732c6ec5a1 done   |
Copying blob 984e9b553bec done   |
Copying blob 36f506d57a55 done   |
Copying blob b4a7ca1217f5 done   |
Copying blob 057ddd0af801 done   |
Copying blob 0e8fc7fd216f done   |
Copying blob 7a9e5ae8d256 done   |
Copying blob 1d75b44bd9c0 done   |
Copying blob 64c6d2e78959 done   |
Copying blob be77d8be8bdf done   |
Copying blob 4e38139f4e33 done   |
Copying blob 6ce2d5edaa3e done   |
Copying blob 34b9ad025475 done   |
Copying blob d2f200484d6f done   |
Copying blob 8d1611934a3f done   |
Copying blob 2fa8e81e1d76 done   |
Copying config 3a121e8852 done   |
Writing manifest to image destination
[INFO] Running filter /usr/local/share/openjournals/data/filters/parse-latex.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/parse-latex.lua in 24 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/inline-cited-references.lua
[INFO] Loaded paper.bib from paper.bib
[INFO] Completed filter /usr/local/share/openjournals/data/filters/inline-cited-references.lua in 56 ms
[INFO] Running filter citeproc
[INFO] Loaded apa.csl from /usr/local/share/openjournals/apa.csl
[INFO] Completed filter citeproc in 187 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/normalize-metadata.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/normalize-metadata.lua in 30 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/time.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/time.lua in 16 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/normalize-author-names.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/normalize-author-names.lua in 25 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/substitute-in-format.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/substitute-in-format.lua in 12 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/resolve-references.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/resolve-references.lua in 40 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/remove-references-heading.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/remove-references-heading.lua in 13 ms
[INFO] Running filter /usr/local/share/openjournals/data/filters/orcid-uri.lua
[INFO] Completed filter /usr/local/share/openjournals/data/filters/orcid-uri.lua in 48 ms
[INFO] Loaded figs/environment.pdf from figs/environment.pdf
[INFO] Loaded figs/workflow.pdf from figs/workflow.pdf
[INFO] Loaded figs/usage.pdf from figs/usage.pdf
pandoc: media: createDirectory: permission denied (Permission denied)
make: *** [Makefile:5: paper] Error 1
[deco]~/src/heudiconv ❱ ls paper.pdf
ls: cannot access 'paper.pdf': No such file or directory
```

--- a/Makefile
+++ b/Makefile
@@ -1,8 +1,17 @@
 OCI_BINARY?=docker
+# For docker, we must specify user so resultant file is not owned by root.
+ifeq ($OCI_BINARY,docker)
+USER_ARG="--user $(id -u):$(id -g)"
+else
+# For Podman, the root user maps to the user running podman.
+# We should not specify a user, it will map to a different UID
+USER_ARG=
+endif
+
 
 .PHONY: paper
 paper: paper.md paper.bib
-	$(OCI_BINARY) run --rm --volume ${PWD}:/data:Z --user $(shell id -u):$(shell id -g) --env JOURNAL=joss docker.io/openjournals/inara
+	$(OCI_BINARY) run $(USER_ARG) --rm --volume ${PWD}:/data:Z --env JOURNAL=joss docker.io/openjournals/inara
 
 .PHONY: clean
 clean:

